docs/content/en/install/eks/_index.md

```diff
@@ -0,0 +1,195 @@
+   b. IAM Role attached to EKS EC2 instances _must_ include permissions to Amazon S3 APIs if you are going to be leveraging backup and restore functionality. See this [IAM policy](https://github.com/k8ssandra/k8ssandra-terraform/blob/ca5b1038d8cee003c0732d5c93ae06592e0a8535/aws/scripts/policy_document.json#L159-L190) as an example. _Note your policy should limit requests to specific buckets and operations._
+2. [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) installed and configured to access the cluster
+3. _Optional_ [Helm](https://helm.sh/docs/intro/install/) installed and configured to access the cluster
+4. _Optional_ [Kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) installed and configured to access the cluster
+
+## Infrastructure Considerations
+
+Apache Cassandra can be deployed across a variety of [hardware profiles](https://cassandra.apache.org/doc/latest/cassandra/operating/hardware.html). Consider the hardware requirements of Apache Cassandra when defining EKS node groups. Determine if you are planning on running a single Cassandra instance per node or multiple instances per node. If you are planning on running multiple instances per node, you will need to ensure that the node has enough resources to support the number of instances you are planning on running. For example, if you are planning on running 3 Cassandra instances per node, you will need to ensure that the node has enough CPU, memory, and storage to support 3 Cassandra instances.
+
+## Dependencies
+
+1. [Cert Manager](https://cert-manager.io/docs/installation/kubernetes/) - Cert Manager provides a common API for management of TLS certificates. It provides interfaces for communicating with a variety of Certificate Authority backends, including self-signed, Venafi, and Vault. K8ssandra Operator uses this API for certificates used by the various K8ssandra components.
+
+## Installation Type
+
+K8ssandra Operator may be deployed in one of two modes. `Control Plane` mode is the default method of installation. In this mode K8ssandra Operator watches for the creation and changes to `K8ssandraCluster` custom resources. When this mode is active it may create Cassandra resources within the local Kubernetes cluster **and / or** remote Kubernetes clusters (in the case of multi-region) deployments. When using K8ssandra Operator you should only have one instance running in `Control Plane` mode. Kubernetes clusters acting as remote regions for Cassandra deployments should be run in `Data Plane` mode. In this mode K8ssandra Operator will not directly reconcile `K8ssandraCluster` resources.
```

Comment:
```suggestion
K8ssandra Operator may be deployed in one of two modes. `Control Plane` mode is the default method of installation. A `Control Plane` instance of K8ssandra Operator watches for the creation and changes to `K8ssandraCluster` custom resources. When `Control Plane` is active Cassandra resources may be created within the local Kubernetes cluster **and / or** remote Kubernetes clusters (in the case of multi-region) deployments. When using K8ssandra Operator you should only have one instance running in `Control Plane` mode. Kubernetes clusters acting as remote regions for Cassandra deployments should be run in `Data Plane` mode. In `Data Plane` mode K8ssandra Operator does not directly reconcile `K8ssandraCluster` resources.
```
should -or must- have one instance of K8ssandra-operator in a Control-Plane node?

Reply:
Across all clusters where k8ssandra-operator is deployed one, and only one, must be configured in control plane mode.

How does this change your suggestion?